src/App.js

```diff
@@ -13,6 +14,7 @@ function App() {
         <div className="App bg" >
           <Navbar></Navbar>
           <Home />
```

Comment:
Put the pokeview component inside of the Home page

Reply:
Done